### PR TITLE
Remove webserver

### LINKED ISSUE #2825: [TECH DEBT] - Remove Webserver and Associated Code
### What is this task and why do we need to work on it?

Kill the webserver code, the webserver examples, the webserver connected network and the `InjectConsensusInfo` hacks all over the code.  

### What work will need to be done to complete this task?

_No response_

### Are there any other details to include?

_No response_

### What are the acceptance criteria to close this issue?

Webserver is gone

### Branch work will be merged to (if not the default branch)

_No response_

### PR DESCRIPTION
**Pull request by ss-es**
Closes #2825

### This PR: 
Deletes all code related to the (old) web server.

### CODE REVIEW

**Review by jparr721 — COMMENTED** [review]
YES

**Review by jparr721 — APPROVED** [review]

File: crates/task-impls/src/consensus/view_change.rs
Comment (jparr721) [inline crates/task-impls/src/consensus/view_change.rs:29]: Not sure if this still violates `too_many_arguments` with the three parameters removed.

File: crates/hotshot/src/traits/networking/libp2p_network.rs
Comment (bfish713) [inline crates/hotshot/src/traits/networking/libp2p_network.rs:1130]: We still need some version of this logic in libp2p (it's a perf optimization but still useful).  Maybe we can roll this logic into `update_view`.  In that function we should start a lookup for the leader 3 views ahead of the new view

File: crates/examples/Cargo.toml
Comment (dailinsubjam) [inline crates/examples/Cargo.toml:1]: Just a note here: push-cdn now use `orchestrator-webserver` as its own orchestrator since they are basically the same, as the interface is now deleted and the code will also be removed in the future (I think?), I create one orchestrator-push-cdn in `push-cdn` folder under my branch `sishan/benchmark_push_cdn_from_main`, the changes should be in when I merge in my branch.

**Review by dailinsubjam — COMMENTED** [review]
Web-server related removal LGTM. Let's wait for others' approval.

**Review by jparr721 — APPROVED** [review]

**Review by jparr721 — DISMISSED** [review]

**Review by bfish713 — APPROVED** [review]

**Review by bfish713 — APPROVED** [review]

**Pull request merged**
